### ISSUE TITLE
[Spree Upgrade][FrontOffice] Fix orders_controller spec, insuficient stock spec (1 broken test)

### ISSUE THREAD

**Issue by luisramos0** [openfoodfoundation/openfoodnetwork#3036]
Fix spec/controllers/spree/orders_controller_spec.rb:135

```
58) Spree::OrdersController viewing cart when an item has insufficient stock displays a flash message when we view the cart
      Failure/Error: expect(flash[:error]).to eq("An item in your cart has become unavailable.")
      
        expected: "An item in your cart has become unavailable."
             got: nil
      
        (compared using ==)
      # ./spec/controllers/spree/orders_controller_spec.rb:135:in `block (4 levels) in <top (required)>'

```

**Comment by luisramos0** [comment]
This is in the same spec, but different case of both #2864 and #3035

**Comment by HugsDaniel** [comment]
@luisramos0 when running this one I get the following : 
![screenshot from 2018-11-22 11-30-32](https://user-images.githubusercontent.com/28786869/48897342-1a3d0f80-ee4a-11e8-8c4b-d514e93af8f0.png)

Which is solved by #3100

**Comment by luisramos0** [comment]
Hey @HugsDaniel you can always check the build for the ultimate state of affairs :-)
https://semaphoreci.com/openfoodfoundation/openfoodnetwork-2/branches/2-0-stable/builds/118
job 3 - test 31:
```

 31) Spree::OrdersController viewing cart when an item has insufficient stock displays a flash message when we view the cart
      Failure/Error: expect(flash[:error]).to eq("An item in your cart has become unavailable.")
      
        expected: "An item in your cart has become unavailable."
             got: "An item in your cart has become unavailable"
      
        (compared using ==)
      # ./spec/controllers/spree/orders_controller_spec.rb:135:in `block (4 levels) in <top (required)>'

```

So, yes, it fixes this issue but there's a typo in the #3100 fix. A missing dot in the end. I fix it in #3125
